controller/examples/examples.go

```diff
@@ -209,6 +210,13 @@ func (e *generator) updateApp() {
 	e.client.UpdateApp(app)
 }
 
+func (e *generator) getAppLog() {
+	res, err := e.client.GetAppLogWithWait(e.resourceIds["app"], 0, false)
+	if err == nil {
+		io.Copy(ioutil.Discard, res)
+	}
+}
```

Comment:
The examples are used in the API docs: https://flynn.io/docs/api/controller#create-app

@jvatic can explain in detail.